src/utils/index.ts

```diff
@@ -378,7 +378,7 @@ export const filterChannelListParams = (params: GroupChannelListQuery, channel:
     return false;
   }
   const searchFilter = params?.searchFilter;
-  if (searchFilter?.query && searchFilter.fields?.length > 0) {
+  if (searchFilter?.query && searchFilter.fields && searchFilter.fields.length > 0) {
```

Comment:
nit: how about this?
```suggestion
  if (searchFilter?.query && (searchFilter?.fields?.length ?? 0) > 0) {
```